internal/suites/example/compose/authelia/resources/entrypoint-backend.sh

```
@@ -2,12 +2,11 @@
 
 set -x
 
-# We move out of the workspace to not include the modules as dependencies of the project.
-cd /
+cd /resources
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)